src/ctapipe/reco/impact.py

```diff
@@ -209,12 +230,32 @@ def __call__(self, event):
         telescope_pointings = self._get_telescope_pointings(event)
 
         # Finally get the telescope images and and the selection masks
-        mask_dict, image_dict, time_dict = {}, {}, {}
+        mask_dict, image_dict, time_dict, ped_dict, spe_dict = {}, {}, {}, {}, {}
         for tel_id in hillas_dict.keys():
             image = event.dl1.tel[tel_id].image
             image_dict[tel_id] = image
             time_dict[tel_id] = event.dl1.tel[tel_id].peak_time
             mask = event.dl1.tel[tel_id].image_mask
+            if self.pedestal_width.tel[tel_id] is None:
+                if event.mon.tel[tel_id].pedestal.charge_std is None:
+                    ped_dict[tel_id] = BACKUP_PED_TABLE[
```

Comment:
I would just remove this fallback (see prev comment), and just require that the user use a proper configuration file.